editor.planx.uk/src/@planx/components/MultipleFileUpload/model.ts

```diff
@@ -82,3 +86,82 @@ export const newFileType = (): FileType => ({
 
 export const checkIfConditionalRule = (condition: Condition) =>
   [Condition.RecommendedIf, Condition.RequiredIf].includes(condition);
+
+export interface UserFiles extends FileType {
+  slot?: FileUploadSlot;
+}
+
+export interface FileList {
+  required: UserFiles[];
+  recommended: UserFiles[];
+  optional: UserFiles[];
+}
+
+export const createFileList = ({
```

Comment:
I'm not totally sold that this is the best place for this logic, but it's good for now!